### PR TITLE
StepanSr: Homework StreamTasks

### PR DESCRIPTION
**Pull request by StepanSrr**

### CODE REVIEW

File: 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        System.out.println();
+        String first =
+                Stream.of("2", 3, "Jack").findFirst().get().toString();
+        System.out.println("print only 2 (first value) -> " + first);
+        System.out.println("///______//////_____");
+
+        // TASK 3
+        // todo print "the quick brown fox and blue hare."
+        String s = "the quick brown fox and the quick blue hare.";
+        Stream.of(s.toLowerCase().split(" ")).distinct().forEach(w -> System.out.print(w + " "));
```
Comment (kykaparov) [inline 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java:50]: The initial text was with "The quick brown fox and the quick blue hare." uppercase T :)
Could you work on that?

File: 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        // TASK 2
+        // todo print only 2
+        // Stream.of("2", 3, "Jack")
+        System.out.print("print only 2 (first 2 values) -> ");
+        Stream.of("2", 3, "Jack").limit(2)
+                .forEach(z -> System.out.print(z + " "));
+        System.out.println();
+        String first =
+                Stream.of("2", 3, "Jack").findFirst().get().toString();
+        System.out.println("print only 2 (first value) -> " + first);
```
Comment (kykaparov) [inline 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java:44]: yeah sorry, by 2 I meant value 2. But it is good that you have solutions for both cases 👍

File: 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java (hunk 1 above)
Comment (StepanSrr) [reply 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java:50]: done

Comment (kykaparov) [reply 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java:50]: I was commenting to others that there is "The" and they have to make it a lower case in the stream. But I see you have done already. 👍

Comment (kykaparov) [reply 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java:50]: One more comment on joining back, can you think how could you not add a space " " after "hare. "?

File: 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java (hunk 2 above)
Comment (StepanSrr) [reply 03-GenericsLambdaStream/src/main/java/homework/Stepan/StreamTasks.java:44]: so id like to keep both then

**Pull request merged**